OsmAnd/src/net/osmand/plus/myplaces/favorites/FavouritesHelper.java

```diff
@@ -686,47 +688,70 @@ private static Comparator<FavouritePoint> getComparator() {
 		};
 	}
 
-	public void updateGroupColor(@NonNull FavoriteGroup group, int color, boolean updatePoints, boolean saveImmediately) {
-		if (updatePoints) {
+	public void updateGroupColor(@NonNull FavoriteGroup group, int color, @NonNull SaveOption saveOption, boolean saveImmediately) {
+		if (APPLY_TO_ALL == saveOption || APPLY_TO_EXISTING == saveOption) {
```

Comment:
could we add boolean methods to SaveOption instead of multiple checks with same code?